tests/integration/conftest.py

```diff
@@ -9,7 +9,7 @@
 
 def pytest_addoption(parser):
     parser.addoption(
-        "--host", action="store", default="dev.indico.io", help="indico ipa host"
+        "--host", action="store", default="dev-ci.us-east-2.indico-dev.indico.io", help="indico ipa host"
```

Comment:
In terms of backwards compatibility, DEVOPS should just setup dev.indico.io CNAME to dev-ci.us-east-2.indico-dev.indico.io

Reply:
@meghanhickey sorry :( I think there's also an issue with the SSL cert not accepting dev.indico.io after all. We might just need to change these.